setup.py

```diff
@@ -22,7 +22,7 @@
     },
     include_package_data=True,
     install_requires=[
-        'django>=1.11,<=3.1',
+        'django>=1.11,<=2.2.19',
```

Comment:
correct me if i'm wrong but doesn't this just restrict client lib on what version is compatible ?
so still we can install 1.11 to 3.1  what advantage is it to restrict it to 2.2.19

Reply:
Ah yeah, you're right - i think i we just change the upper limit to the latest secure 3.1.x -> 3.1.7

Reply:
```suggestion
        'django>=1.11,<=3.1.7',
```

How about this instead @webbyfox  @depsiatwal

Reply:
I guess this is just to make the Github alert bot happy. the actual installation will not force to go higher version if there is already Django exist. happy with this.